package/sbp_fileio_daemon/sbp_fileio_daemon/src/sbp_fileio.c

```diff
@@ -256,12 +710,13 @@ static void remove_cb(u16 sender_id, u8 len, u8 msg[], void *context)
 static void write_cb(u16 sender_id, u8 len, u8 msg_[], void *context)
 {
   (void)sender_id;
+  u8 headerlen = 0;
```

Comment:
This definition could be closer to the place of use like other variables.

Reply:
👍